fancy_collections/formatting.py

```diff
@@ -1,8 +1,10 @@
 #!/usr/bin/env python
 from __future__ import annotations
 
+import shutil
 from typing import Tuple, List, Iterable, Any
 
+from matplotlib import shutil
```

Comment:
`shutil` is part of the python stdlib. 
if for some reason you really need `shutil` from `matplotlib`, i must reject this PR, because this brings a much to heavy dependency to this lightweight lib. :shrug: :grimacing:

Reply:
No...That was just the auto import feature failing.
```suggestion
import shutil
```